x-pack/plugins/ingest_manager/server/services/epm/registry/index.ts

```diff
@@ -50,7 +50,7 @@ export async function fetchFindLatestPackage(
   const res = await fetchUrl(url.toString());
   const searchResults = JSON.parse(res);
   if (searchResults.length) {
-    return searchResults[0];
+    return searchResults[searchResults.length - 1];
```

Comment:
Are we sure that the results will always be ordered by version?

Reply:
good question.  no.  i think i was assuming based on the api doc that search returns "the most recent packages available", but i'm not exactly sure what that means. @ruflin can you confirm how the output is sorted?  I can always sort it on my end if for whatever reason we don't want to sort it by version in the registry api.

Reply:
if it's not sure I will just add a sort by version before returning, if it can avoid a bug later

Reply:
The order is currently alphabetical. So this does not help here. There is no guarantee at the moment on the order for this :-(

Reply:
@nchaulet @ruflin okay i've added sorting